src/Categories/Bicategory/Construction/LaxSlice.agda

```diff
@@ -0,0 +1,245 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- Mentioned in passing https://ncatlab.org/nlab/show/slice+2-category
+
+open import Categories.Bicategory using (Bicategory)
+
+module Categories.Bicategory.Construction.LaxSlice
+       {o ℓ e t}
+       (𝒞 : Bicategory o ℓ e t)
+       where
+
+open import Categories.Category using () renaming (Category to 1Category)
+import Categories.Morphism.Reasoning as MR
+open import Categories.Bicategory.Extras 𝒞
+open Shorthands
+
+open import Categories.Functor.Construction.Constant using (const)
+open import Categories.Functor.Bifunctor using (Bifunctor)
+open import Categories.NaturalTransformation.NaturalIsomorphism using (NaturalIsomorphism; niHelper)
+open import Function using (_$_)
+open import Data.Product using (_,_)
+open import Categories.Functor using (Functor)
+open Functor using (F₀)
+
+open import Level using (_⊔_)
+
+record SliceObj (X : Obj) : Set (t ⊔ o) where
+  constructor sliceobj
+  field
+    {Y} : Obj
+    arr : Y ⇒₁ X
+
+module SliceHom (A : Obj) where
+
+  record Slice⇒₁ (X Y : SliceObj A) : Set (o ⊔ ℓ) where
+    constructor slicearr₁
+    private
+      module X = SliceObj X
+      module Y = SliceObj Y
+
+    field
+      {h} : X.Y ⇒₁ Y.Y
+      Δ   : X.arr ⇒₂ (Y.arr ∘ₕ h)
+
+  record Slice⇒₂ {X Y : SliceObj A} (J K : Slice⇒₁ X Y) : Set (ℓ ⊔ e) where
+    constructor slicearr₂
+    private
+      module Y = SliceObj Y
+      module J = Slice⇒₁ J
+      module K = Slice⇒₁ K
+
+    field
+      {ϕ} : J.h ⇒₂ K.h
+      E   : K.Δ ≈ (Y.arr ▷ ϕ ∘ᵥ J.Δ)
+
+  _∘ᵥ/_ : ∀ {X Y : SliceObj A}{J K L : Slice⇒₁ X Y} → Slice⇒₂ K L → Slice⇒₂ J K → Slice⇒₂ J L
+  _∘ᵥ/_ {X}{Y}{J}{K}{L} (slicearr₂ {ϕ = ϕ} E) (slicearr₂ {ϕ = ψ} F) = slicearr₂ {ϕ = ϕ ∘ᵥ ψ} $ begin
+      L.Δ                             ≈⟨ E ⟩
+      (Y.arr ▷ ϕ ∘ᵥ K.Δ)              ≈⟨ refl⟩∘⟨ F ⟩
+      Y.arr ▷ ϕ ∘ᵥ (Y.arr ▷ ψ ∘ᵥ J.Δ) ≈⟨ pullˡ ∘ᵥ-distr-▷ ⟩
+      Y.arr ▷ (ϕ ∘ᵥ ψ) ∘ᵥ J.Δ         ∎
+    where module X = SliceObj X
+          module Y = SliceObj Y
+          module J = Slice⇒₁ J
+          module K = Slice⇒₁ K
+          module L = Slice⇒₁ L
+          open 1Category (hom X.Y A)
+          open HomReasoning
+          open Equiv
+          open MR (hom X.Y A)
+          
+  SliceHomCat : SliceObj A → SliceObj A → 1Category (o ⊔ ℓ) (ℓ ⊔ e) e
+  SliceHomCat X Y = record
+    { Obj = Slice⇒₁ X Y
+    ; _⇒_ = Slice⇒₂
+    ; _≈_ = λ (slicearr₂ {ϕ} _) (slicearr₂ {ψ} _) → ϕ ≈ ψ 
+    ; id = slice-id _
+    ; _∘_ = _∘ᵥ/_
+    ; assoc = hom.assoc
+    ; sym-assoc = hom.sym-assoc
+    ; identityˡ = hom.identityˡ
+    ; identityʳ = hom.identityʳ
+    ; identity² = hom.identity²
+    ; equiv = record
```

Comment:
When it's that short, I tend to put such `equiv` records all on one line. Your choice.